MH4U Dex/HomeViewController.m

```diff
@@ -28,6 +30,21 @@ - (IBAction)resetButtonPressed:(id)sender
     [CoreDataController setShouldTriggerReloadUponRestart:YES];
 }
 
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    [MBProgressHUD showHUDAddedTo:self.view animated:YES];
+    dispatch_queue_t queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
+    dispatch_async(queue, ^{
+        [[CoreDataController sharedCDController] tryLoadSequence];
```

Comment:
Should I make a backgroundObjectContext property and, when it is created, create it in a dispatch_async block?  The issue with that is, how would I then limit it to one specific background thread?